packages/babel-core/src/config/full.ts

```diff
@@ -490,11 +490,11 @@ const instantiatePreset = makeWeakCacheSync(
   },
 );
 
-function chain(a, b) {
+function chain(a: any, b: any) {
```

Comment:
```suggestion
function chain<Args extends any[])(a: undefined | (...args: Args) => void, b: undefined | (...args: Args) => void) {
```

and use Args instead of unknown[] below.